app/src/main/java/org/httpsknighthacks/knighthacksandroid/FAQs.java

```diff
@@ -57,15 +57,15 @@ public void onStart() {
                 mProgressBar.setVisibility(View.VISIBLE);
             }
 
-            @Override
+             @Override
             public void onSuccess(ArrayList<FAQ> response) {
                 int numFAQs = response.size();
                 for (int i = 0; i < numFAQs; i++) {
                     FAQ currFAQ = response.get(i);
 
                     if (FAQ.isValid(currFAQ)) {
-                        mCardTitleList.add(currFAQ.getQuestionOptional().getValue());
-                        mCardDetailsList.add(currFAQ.getAnswerOptional().getValue());
+                        mCardTitleList.add(currFAQ.getQuestion());
+                        mCardDetailsList.add(currFAQ.getAnswer());
```

Comment:
I'm curious. Why was the optional logic here changed?

Reply:
It was possible to keep the optionals, but they aren't necessary for our use case. All we were using it for was checking if a value exists before operating on it. It just changes the semantics from if (value == null) to if (optionalValue.isPresent()). There are other ways of using it that would justify its use but it's not something I believe we need to include.